src/lib/index.ts

```diff
@@ -204,11 +203,7 @@ const generateIconFromSVG = async (
   logger.log('  src: ' + svgFilePath)
   logger.log('  dir: ' + destDirPath)
 
-  const workDir = path.join(os.tmpdir(), uuidv4())
-  fs.mkdirSync(workDir)
-  if (!fs.existsSync(workDir)) {
-    throw new Error('Failed to create the working directory.')
-  }
+  const workDir = fs.mkdtempSync('tmp-xxxxxx')
```

Comment:
```suggestion
  const workDir = fs.mkdtempSync('icon-gen-')
```

https://nodejs.org/api/fs.html#fspromisesmkdtempprefix-options
> Due to platform inconsistencies, avoid trailing X characters in prefix. Some platforms, notably the BSDs, can return more than six random characters, and replace trailing X characters in prefix with random characters.

The reference notes that `x` seems to be a placeholder for a specific platform. Therefore, it seems that it is only necessary to use the automatic suffix.

As for the prefix, I used `icon-gen` because I want to make it clear that this is a directory dedicated to this tool. This will also contribute to uniqueness.